contracts/LosslessReporting.sol

```diff
@@ -117,29 +123,25 @@ contract LosslessReporting is Initializable, ContextUpgradeable, PausableUpgrade
 
     /// @notice This function sets the default reporter reward
     /// @param reward Percentage rewarded to the reporter when a report gets resolved positively
-    function setReporterReward(uint256 reward) public onlyLosslessAdmin {
-        require(reward + losslessReward + committeeReward + stakersReward <= 100, "LSS: Total exceed 100");
+    function setReporterReward(uint256 reward) public onlyLosslessAdmin cannotExceedHundred {
```

Comment:
The modifier is actually running after the new value is set because the "_;" is at the beginning. What it does is first assign the new value and then run the modifier. 

![image](https://user-images.githubusercontent.com/13722088/150992774-ee91ced6-299a-45da-b5ea-d0417ea1d6c9.png)